enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+|-------------------------|---------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| unpaused                | unpaused                  | unpaused                    | Traditional, fully self-managed change rollouts. Material changes can be initiated immediately upon configuration change. |
+| paused (any strategy)   | **unpaused**              | **paused**                  | Changes to both the control-plane and worker-nodes are constrained by the control-plane strategy. |
+| unpaused                | paused (any strategy)     | paused                      | Material changes can be initiated immediately on the control-plane. Material changes on worker-nodes are subject to the worker-node policy. |
+| paused (any strategy)   | paused (any strategy)     | paused                      | Material changes to the control-plane are subject to change control strategy for the control-plane. Material changes to the worker-nodes are subject to **both** the control-plane and worker-node strategies - if either precludes material change initiation, changes are left pending. |
+
+#### Maintenance Schedule Strategy
+The maintenance schedule strategy is supported by all resources which support change management. The strategy
+is configured by specifying an RRULE identifying permissive datetimes during which material changes can be
+initiated. The cluster lifecycle administrator can also exclude specific date ranges, during which 
+material changes will be paused.
+
+#### Disabled Strategy
+This strategy indicates that no change management strategy is being enforced by the resource. It always implies that
+the enforcement state at the resource level is unpaused / permissive. This does not always
+mean that material changes are permitted due to change management hierarchies. For example, a MachineConfigPool
+with `strategy: Disabled` would still be subject to a `strategy: MaintenanceStrategy` in the ClusterVersion resource.
+
+#### Assisted Strategy - MachineConfigPool
+Minimally, this strategy will be supported by MachineConfigPool. If and when the strategy is supported by other
+change management capable resources, the configuration schema for the policy may differ as the details of
+what constitutes and informs change varies between resources. 
+
+This strategy is motivated by the desire to support the separation of control-plane and worker-node updates both
+conceptually for users and in real technical terms. One way to do this for users who do not benefit from the
+`MaintenanceSchedule` strategy is to ask them to initiate, pause, and resume the rollout of material
+changes to their worker nodes. Contrast this with the fully self-managed state today, where worker-nodes
+(normally) begin to be updated automatically and directly after the control-plane update.

```

Comment:
If I understood correctly, we are stating here how MCO works today. If that's the case, I think we want to say here control-plane and worker node starts update in parallel today based on maxUnavailable value set for the respective MachineConfigPool.

Reply:
ack. Clarifying in the next push that the update is subject to `maxUnavailable`.